spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt

```diff
@@ -1438,16 +1442,13 @@ private fun ColorPreview(
                     ColorItem(SparkTheme.colors.supportVariant, "support Variant")
                 }
                 Row {
-                    ColorItem(SparkTheme.colors.background, "background")
```

Comment:
nit: all these could be rewritten with a simpler KProperty access to avoid potenial typos 🙈

Reply:
```kotlin
fun ColorItem(color: KProperty0<Color>)
```

Reply:
Then we create a `List<List<Color>>` and iterate over it to build the Composable tree.

Reply:
Something like:

```kotlin
val colors = with(SparkTheme.colors) {
    listOf(
        listOf(
            listOf(::main, ::mainContainer, ::mainVariant),
            listOf(::support, ::supportContainer, ::supportVariant),
            listOf(::accent, ::accentContainer, ::accentVariant),
            listOf(::basic, ::basicContainer),
            listOf(::support, ::supportContainer, ::supportVariant),
            listOf(::background, ::surface, ::surfaceInverse),
            listOf(::outline, ::outlineHigh),
        ),
        listOf(
            listOf(::success, ::successContainer),
            listOf(::alert, ::alertContainer),
            listOf(::error, ::errorContainer),
            listOf(::info, ::infoContainer),
            listOf(::neutral, ::neutralContainer),
        ),
    )
}

colors.forEach {
    Column {
        it.forEach {
            Row {
                it.forEach(::ColorItem)
            }
        }
    }
}
```

Reply:
Not here, I'll update it in #660 with the example